__tests__/app.test.js

```diff
@@ -68,3 +68,69 @@ describe("Get specific article ID", () => {
       });
   });
 });
+
+describe.only("PATCH /api/articles/:article_id ", () => {
+  test("Status 200 : responds with status 200 ", () => {
+    return request(app)
+      .patch("/api/article/3")
+      .send({ inc_votes: 1 })
+      .expect(200);
+  });
+  test("Status 200 : responds with an object of the updated article ", () => {
+    return request(app)
+      .patch("/api/article/3")
+      .send({ inc_votes: 1 })
+      .expect(200)
+      .then(({ body }) => {
+        expect(body).toEqual({
+          updatedArticle: {
+            article_id: 3,
+            title: "Eight pug gifs that remind me of mitch",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "some gifs",
+            created_at: "2020-11-03T09:12:00.000Z",
+            votes: 1,
+          },
+        });
+      });
+  });
+  test("Status 200 : responds with an object of the updated article -- SECOND TEST ", () => {
```

Comment:
I'd rename this test to be more specific, maybe something like 'responds with an object of the updated article when passed a negative inc_votes value' or if you can find a more concise way of explaining that you're checking it works with this negative number 😄